### PR TITLE
Reehah's stage 2 task

### PR DESCRIPTION
**Pull request by Reehah**
my HNGi7 stage 2 task

### CODE REVIEW

**Comment by Marvellous1** [comment]
Add reehah please include email to the message in this format Hello World, this is ${fullname} with HNGi7 ID ${id} and email ${email} using ${language} for stage 2 task

And also, assign the two other to a variable. You are only assigning the first one ie 
var name = "";
var id = "";

File: scripts/godstime.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+var name="Alade-Raji Far'hah";
+id = 'HNG-00449';
+lang = 'node.js';
+
+console.log('Hello World, this is', name,'with HNGi7 ID', id,'using',lang,'for stage 2 task');
```
Comment (BenjaminKyamanywa) [inline scripts/godstime.js]: hey @Reehah, somethings to look at for review. add 'var' to all your variable before you initialize them for consistency. also, add a variable for your email and incorporate it into your final output. 
The output should follow this; 
**Hello World, this is [full name] with HNGi7 ID [ID] and email [email] using [language] for stage 2 task**
ps: everything between the stars is what you want to output the console.

With the above edits all should be well.

**Comment by Reehah** [comment]
Okay. Thank you

**Comment by Reehah** [comment]
Thank you very much. I'll do that before the end of the day.

**Comment by BenjaminKyamanywa** [comment]
You might want to make the changes as soon as you can rather than later to avoid any surprises. Your welcome. Also, you will want to make a pull request from a new branch that you create from your repo other than your master branch ie make the pull request from a new branch that you create and not from your master branch.

**Comment by Reehah** [comment]
Please explain in simpler terms. I'm new to this. Thank you

**Comment by BenjaminKyamanywa** [comment]
hey, you might want to watch this when you have the time later in your free time
https://www.youtube.com/watch?v=SWYqp7iY_Tc

Otherwise, follow the instructions from this slack message but in regard to this current repository as they are instructions from the other old team leaders repository that we were using initially. Here;

https://hngi7.slack.com/archives/C01482Z0095/p1591278934391300

all git workflow details are there for making a new branch from your forked repository and making a pull request from the new branch.
